hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -30,20 +34,22 @@
 
 import java.io.IOException;
 import java.io.UncheckedIOException;
+import java.net.BindException;
 import java.net.InetSocketAddress;
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+import static com.hazelcast.internal.util.AddressUtil.transformPortDefinitionsToPorts;
 import static java.lang.System.getProperty;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 @SuppressWarnings("checkstyle:MagicNumber, checkstyle:")
 public class TpcServerBootstrap {
```

Comment:
Perhaps we should rename this class to AltoBootstrap.

Reply:
Not related to this PR. I will open a really small separate PR for that. WDYT?

Also as a second note, this alto/tpc differentiation became a little confusing. Why don't we select one and always use that everywhere. Even if TPC is the engine of the Alto, we won't have a second engine so we can just call it AltoEngine for example.

Reply:
NP